riff-raff.yaml

```diff
@@ -12,7 +12,7 @@ deployments:
     parameters:
       amiTags:
         BuiltBy: amigo
-        Recipe: editorial-tools-focal-java8-ARM-WITH-cdk-base
+        Recipe: editorial-tools-focal-java11-ARM-WITH-cdk-base
```

Comment:
Here we're changing AMIgo recipes, from:

[`editorial-tools-focal-java8-ARM-WITH-cdk-base`](https://amigo.gutools.co.uk/recipes/editorial-tools-focal-java8-ARM-WITH-cdk-base)

...to:

[`editorial-tools-focal-java11-ARM-WITH-cdk-base`](https://amigo.gutools.co.uk/recipes/editorial-tools-focal-java11-ARM-WITH-cdk-base)

The newer recipe already exists, and is [in use](https://amigo.gutools.co.uk/recipes/editorial-tools-focal-java11-ARM-WITH-cdk-base/usages) for Composer, editorial feeds, etc.